testing/ostest/signest.c

```diff
@@ -295,11 +298,12 @@ void signest_test(void)
 
   /* Start interfering thread  */
 
-  prio++;
-  printf("signest_test: Starting interfering task at priority %d\n", prio);
-  interferepid = task_create("interfere", prio, STACKSIZE,
```

Comment:
let's change task_create to posix_spawn

Reply:
OK. That requires quite a bit of extra work to change with posix_spawn for kernel mode. 
1. some case use many global variables to sync between tasks, but global variable is not the same copy between process for kernel mode
https://github.com/apache/nuttx-apps/blob/master/testing/ostest/signest.c#L56
2. waitpid to check WEXISTATUS is wrong in kernel mode.